awd/cli.py

```diff
@@ -301,3 +303,39 @@ def listen(
         logger.debug(f"Logs sent to {ctx.obj['log_recipient_email']}")
     except ClientError as e:
         logger.error(f"Failed to send logs: {e.response['Error']['Message']}")
+
+
+@cli.command()
+def check_permissions():
```

Comment:
It seems that there's nothing specific to AWD in this? Does the same code exist in DSS?